docs/authentication.rst

```diff
@@ -320,8 +320,8 @@ each time to update the session cookie on expiration:
 
 * Improved security via JWT-based session tokens that can only be generated using authorized service accounts.
 * Stateless session cookies that come with all the benefit of using JWTs for authentication. The session cookie has
-  the same claims (including custom claims) as the ID token, making the same permissions checks enforceable on the
-  session cookies.
+    the same claims (including custom claims) as the ID token, making the same permissions checks enforceable on the
+    session cookies.
```

Comment:
Oh no!
This makes this
![image](https://github.com/kreait/firebase-php/assets/952007/89aa690e-4a36-41ce-841b-29257add23d2)
into this
![image](https://github.com/kreait/firebase-php/assets/952007/a8d32b95-9821-45d3-ae34-34e135cf36d3)

Reply:
Should rst and md files be indented with 2 spaces?